locales/en.ini

```diff
@@ -13,6 +13,7 @@ qa-intro = So you enjoy finding other programmers' mistakes?
 webqa-extra = find problems in Mozilla's websites
 ffqa-extra = help us find bugs in Firefox
 thunderbirdqa-extra = help us find bugs in Thunderbird
+oneanddone-extra = learn about Mozilla QA
```

Comment:
agreed- I'll ponder on how to better describe a tool that covers all types of QA tasks